src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -6,11 +6,12 @@ import io.circe.generic.extras.Configuration
 import io.circe.generic.extras.auto._
 import org.scalatest.Assertion
 import org.scalatest.matchers.should.Matchers
-import uk.gov.nationalarchives.tdr.api.service.FileMetadataService.{FileReference, FileType, Filename, ParentReference, clientSideProperties, serverSideProperties}
+import uk.gov.nationalarchives.tdr.api.service.FileMetadataService.{FileReference, FileType, Filename, ParentReference, clientSideProperties}
```

Comment:
You don't need this import. 

All of these classes are imported with this: `import uk.gov.nationalarchives.tdr.api.service.FileMetadataService._` on line 14

See here for some background on import clauses in Scala, if you not already taken a look: https://docs.scala-lang.org/tour/packages-and-imports.html#imports